libobs/util/base.c

```diff
@@ -20,11 +20,7 @@
 #include "c99defs.h"
 #include "base.h"
 
-#ifdef _DEBUG
 static int log_output_level = LOG_DEBUG;
-#else
-static int log_output_level = LOG_INFO;
-#endif
```

Comment:
I left it because I envisioned someone wanting to change that sometime in the future. Just seemed like something that shouldn't be buried in the functions.